docs/slimmer_templates.md

```diff
@@ -10,12 +10,20 @@ This is the same as the `gem_layout` template, except that this layout **doesn't
 
 Use this layout if you want to have full width content - such as the blue welcome bar on the GOV.UK homepage.
 
-## `gem_layout_account`
+## `gem_layout_account` (deprecated – use `gem_layout_account_manager` instead)
```

Comment:
Presumably something is still using this layout, which is why it can't be removed/replaced?

Reply:
Yes you're correct, [`frontend` is still using this layout ](https://github.com/alphagov/frontend/blob/465a00aae52b6bd762debc14a0f1aba3b61d8dd4/app/controllers/account_home_controller.rb#L3). Once that page in `frontend` has been updated to the other layout (which should be a fairly straightforward ordeal), this one can be removed.

I'm going to note that in this doc to make it clearer.

Reply:
Have updated the doc with more detail as to where this template is still being used, and plans to remove it.